### DISCUSSION TITLE
Cover art for non-graphical terminals

### DISCUSSION THREAD

**Discussion by rogerfarrell** [jayadamsmorgan/Yatoro#26] [Q&A]
This looks great! How do you plan on handling cover art for terminal emulators that do not support images?

**Comment by jayadamsmorgan** [comment]
Hi, thanks for your feedback! That's a good question. The artwork is drawn with pixels if terminal supports it, otherwise it should fallback to drawing with braille dots or with cell parts (halves, quadrants, sextants, etc.)

However, I did not test Yatoro with terminal emulators that do not support drawing with pixels yet. This definitely has to be addressed, thanks!

**Reply by cooperwhitley** [reply]
Hi! Loving this so far, but in kitty term images are not rendered with pixels despite the terminal being capable of this. Do you know why this may be?

**Reply by jayadamsmorgan** [reply]
> Hi! Loving this so far, but in kitty term images are not rendered with pixels despite the terminal being capable of this. Do you know why this may be?

Hi @cooperwhitley ! Thanks for your interest in Yatoro! Just made a new release where you can force the way artworks are rendered through `ui.artwork.blit` config option. You can read more about it in [CONFIGURATION.md](https://github.com/jayadamsmorgan/Yatoro/blob/master/CONFIGURATION.md).

Upgrade with `brew update` and `brew upgrade yatoro`

**Reply by cooperwhitley** [reply]
Awesome! Displaying perfectly now.
